Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -0,0 +1,53 @@
+function Add-Tls12InSession {
+    [CmdletBinding()]
+    param()
+
+    try {
+        if ([Net.ServicePointManager]::SecurityProtocol -notcontains 'Tls12') {
+            $securityProtocol=@()
+            $securityProtocol+=[Net.ServicePointManager]::SecurityProtocol
+            $securityProtocol+=[Net.SecurityProtocolType]3072
```

Comment:
use [System.Net.SecurityProtocolType]::Tls12 here instead of hard coding 3072

Reply:
[System.Net.SecurityProtocolType]::Tls12 enum is not present in .Net 4.0. So the recommendation is to use 3072 which works everywhere